lib/cog/template_cache.ex

```diff
@@ -1,7 +1,6 @@
 defmodule Cog.TemplateCache do
```

Comment:
This `TemplateCache` is really just a GenServer wrapper around an ets table. I wasn't happy with tangling this all up with the `Template` module (which isn't even it's own process) so it seemed reasonable to just throw the cache in its own process. The only downside is that it takes 2 message sends to lookup and then insert, where before we were able to do that in one call. Feedback on this decision would be great though.